src/main/webapp/play.html

```diff
@@ -5,12 +5,12 @@
 <title>Ant Media Server WebRTC/HLS Player</title>
 
 <!-- videojs includes  -->
-<link href="https://unpkg.com/video.js@7.8.3/dist/video-js.css" rel="stylesheet">
+  <link href="https://unpkg.com/video.js@7.8.4/dist/video-js.css" rel="stylesheet">
 
-<script src="https://unpkg.com/video.js@7.8.3/dist/video.js"></script>
+  <script src="https://vjs.zencdn.net/7.8.4/video.js"></script>
+  <script src="https://unpkg.com/videojs-contrib-hls.js"></script>
```

Comment:
We need to use videojs-http-streaming. videojs-contrib-hls.js is obsolete
https://github.com/videojs/http-streaming